src/Layout/index.jsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import { gtaModel } from '../constants/gtaModel';
-import { GtaVersions } from '../Components';
+import { gtaData } from '../data/data';
+import { GtaVersions } from '../components/gtaVersions/index';
```

Comment:
remove index. When importing files, if you are importing from the folder which has index.js/ts, you can just write the folder name.